packages/contracts/contracts/test/Colosseum.t.sol

```diff
@@ -1267,7 +1267,7 @@ contract Colosseum_ValidatorSystemUpgrade_Test is Colosseum_Initializer {
         return arr;
     }
 
-    function _getFirstSegments() private view returns (bytes32[] memory) {
+    function _getFirstSegments() internal view returns (bytes32[] memory) {
```

Comment:
Why is the visibility of this function changed?

Reply:
Same as the answer above